code/zoltar_scripts/remove_covid19_forecasts_from_zoltar.py

```diff
@@ -0,0 +1,51 @@
+from zoltpy import util
```

Comment:
Does this script basically _sync_ the GitHub repo with Zoltar? I think @nickreich had mentioned once about not doing this. Basically, all I know is we should not delete any extra forecast files from Zoltar unless its a duplicate of another one. What do you think, @nickreich ?

Reply:
But also on this [thread](https://github.com/reichlab/covid19-forecast-hub/issues/405) @nickreich said "there some forecast that didn't have any valid rows remaining once we removed the day ahead death targets", do we want to remove these also?